actors/builtin/miner/miner_actor.go

```diff
@@ -1334,6 +1341,7 @@ func checkPrecommitExpiry(rt Runtime, sectors *abi.BitField) {
 }
 
 // TODO: red flag that this method is potentially super expensive
+// https://github.com/filecoin-project/specs-actors/issues/411
```

Comment:
This one deserves a distinct issue for the fact that it loads all the deal IDs into a slice and makes a method call with it to terminate them.